packages/node-cli/src/TokenStore.ts

```diff
@@ -14,37 +16,84 @@ import { TokenResponse } from "@openid/appauth";
 export class TokenStore {
   private readonly _appStorageKey: string;
   private readonly _scopes: string;
-
-  public constructor(namedArgs: {clientId: string, issuerUrl: string, scopes: string}) {
+  private readonly _store: NodePersist.LocalStorage;
+  public constructor(namedArgs: {clientId: string, issuerUrl: string, scopes: string}, dir?: string) {
     // A stored credential is only valid for a combination of the clientId, the issuing authority and the requested scopes.
     // We make the storage key a combination of clientId and issuing authority so that keys can stay cached when switching
     // between PROD and QA environments.
     // We store the scopes in our password blob so we know if a new token is required due to updated scopes.
-    this._appStorageKey = `iTwinJs:${namedArgs.clientId}:${namedArgs.issuerUrl}`;
+    const configFileName = `itwinJs_${namedArgs.clientId}`;
```

Comment:
```suggestion
    const configFileName = `iTwinJs_${namedArgs.clientId}`;
```

is this the same structure as the electron auth token?